compiler-rt/lib/scudo/standalone/allocator_config.def

```diff
@@ -0,0 +1,123 @@
+//===-- allocator_config.def ------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines all the flags and types supported in Scudo.
+
+#ifndef BASE_REQUIRED_TEMPLATE_TYPE
```

Comment:
I'm not sure about this suggestion, but should all of these be prefaced with CONFIG_XXX? It might make the names too long, but it would be explicit that this is specifically configuration related.

Reply:
Good question. I thought of this suggestion a little bit. I agree that the prefix will make it more clear but the trade-off is the length. Given that these macros are only used in allocator_config.def and allocator_config_wrapper.h which explicitly indicate they are something for the configuration, I may prefer keeping it like this for now. 

Later soon, we will also be using these macros for other purposes (like dumping the allocator config). I'll see if the names become ambiguity (and the filename can't help with that) then do the change when needed.